api/client/node/ps.go

```diff
@@ -31,6 +32,7 @@ func newPSCommand(dockerCli *client.DockerCli) *cobra.Command {
 		},
 	}
 	flags := cmd.Flags()
+	flags.BoolVar(&opts.noTrunc, "no-trunc", false, "Don't truncate output")
```

Comment:
nit: perhaps change to `Do not` (I know we do it in the other output, so not your fault)